src/DependencyInjection/UiElementRegistryPass.php

```diff
@@ -75,7 +75,9 @@ public function process(ContainerBuilder $container): void
      */
     private function validateUiElementResource(string $class): void
     {
-        if (!\in_array(UiElementInterface::class, class_implements($class), true)) {
+        $interfaces = (array) (class_implements($class) ?? []);
```

Comment:
Why this change ?
It seems that this methods always returns an array : https://www.php.net/manual/fr/function.class-implements.php

Reply:
https://www.php.net/manual/fr/function.class-implements.php#refsect1-function.class-implements-returnvalues
says "Retourne un tableau ou false en cas d'erreur."

also in phpdoc: 
```
/**
 * Return the interfaces which are implemented by the given class
 * @link https://php.net/manual/en/function.class-implements.php
 * @param object|string $object_or_class <p>
 * An object (class instance) or a string (class name).
 * </p>
 * @param bool $autoload [optional] <p>
 * Whether to allow this function to load the class automatically through
 * the __autoload magic
 * method.
 * </p>
 * @return string[]|false An array on success, or false on error.
 */
 ```
the in_array require an array as second parameter: 
https://www.php.net/manual/fr/function.in-array.php

and phpstan trigger an error without this
```
vendor/bin/phpstan analyse -c phpstan.neon src                                                                                                                                                      git:(sylius1.10|) 
 32/32 [▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓] 100%

 ------ ----------------------------------------------------------------------------------------------- 
  Line   DependencyInjection/UiElementRegistryPass.php                                                  
 ------ ----------------------------------------------------------------------------------------------- 
  80     Parameter #2 $haystack of function in_array expects array, array<string, string>|false given.  
 ------ ----------------------------------------------------------------------------------------------- 

                                                                                                                        
 [ERROR] Found 1 error                                                                                                  
```